docs/documentation/guides/node.mdx

```diff
@@ -5,7 +5,7 @@ title: "Node"
 This guide demonstrates how to use Infisical to manage secrets for your Node stack from local development to production. It uses:
 
 - Infisical (you can use [Infisical Cloud](https://app.infisical.com) or a [self-hosted instance of Infisical](https://infisical.com/docs/self-hosting/overview)) to store your secrets.
-- The [infisical-node](https://github.com/Infisical/infisical-node) client SDK to fetch secrets back to your Node application on demand.
+- The [infisical-node](https://infisical.com/docs/sdks/languages/node) client SDK to fetch secrets back to your Node application on demand.
```

Comment:
We'll need to update the rest of the guide vlad. The rest of the guide follows service tokens which are not compatible with new sdks. 

At @DanielHougaard can update this after break so that it is more consistent with new sdk steps. If you want to merge this in, i would revert the link back to old sdk link to reduce confusion. What do you think?